bindings/azure/servicebusqueues/metadata.yaml

```diff
@@ -167,3 +158,45 @@ metadata:
     default: '60'
     example: '30'
     # binding is not specified so it's the same as <root>.binding
+  - name: disableEntityManagement
+    description: "When set to true, queues and subscriptions do not get created automatically. Default: 'false'"
+    type: bool
+    default: 'false'
+    example: 'true'
+  - name: lockDurationInSec
+    description: "Defines the length in seconds that a message will be locked for before expiring. Used during subscription creation only. Default set by server."
+    type: number
+    example: '30'
+  - name: autoDeleteOnIdleInSec
+    description: "Time in seconds to wait before auto deleting idle subscriptions. Used during subscription creation only. Default: `0` (disabled)"
+    default: '0'
+    type: number
+    example: '3600'
+  - name: defaultMessageTimeToLiveInSec
+    description: "Default message time to live, in seconds. Used during subscription creation only."
+    type: number
+    example: '10'
+  - name: maxDeliveryCount
+    description: "Defines the number of attempts the server will make to deliver a message. Used during subscription creation only. Default set by server."
+    type: number
+    example: '10'
+  - name: handlerTimeoutInSec
+    description: "Timeout for invoking the app's handler. Default: `0`"
+    type: number
+    example: "30"
+    default: "0"
+  - name: publishMaxRetries
+    description: 'The max number of retries for when Azure Service Bus responds with "too busy" in order to throttle messages. Defaults: `5`'
+    type: number
+    example: "10"
+    default: "5"
+    binding:
+      output: true
+  - name: publishInitialRetryIntervalInMs
```

Comment:
The docs seem to be wrongly saying `publishInitialRetryInternalInMs` (https://docs.dapr.io/reference/components-reference/supported-bindings/servicebusqueues/)

Reply:
I think the docs were not updated yet - I compared this against an open docs PR at the time.